vnext/Microsoft.ReactNative/ABICxxModule.cpp

```diff
@@ -58,14 +58,14 @@ std::vector<CxxModule::Method> ABICxxModule::getMethods() {
 void ABICxxModule::InitEvents(std::vector<ABICxxModuleEventHandlerSetter> const &eventHandlerSetters) noexcept {
   for (auto &eventHandler : eventHandlerSetters) {
     eventHandler.EventHandlerSetter([ this, name = eventHandler.Name ](ReactArgWriter const &argWriter) noexcept {
-      DynamicWriter writer;
+      auto writer = make<DynamicWriter>();
```

Comment:
>auto [](start = 6, length = 4)

(nit) why do we need this change?

Reply:
It seems that previously this code was creating classes on the call stack and the new check causes a compilation error. Our tests are passing because they do not asynchronous code inside. Though customer code would fail if they ever do.

---
In reply to: [376125176](https://github.com/microsoft/react-native-windows/pull/4045#discussion_r376125176) [](ancestors = 376125176)